popup.js

```diff
@@ -85,6 +88,7 @@ function onFontsLoad(json, fromCache) {
     lscache.set(LS_FONTS_API, json, 60*12);
   }
   loadVisibleFonts();
+  onSubsetChange();
```

Comment:
One consequence of the current code flow is that for the initial load, it ends up re-storing the two localstorage values unnecessarily and calling loadVisibleFonts three times. An alternate approach is to pass an isInitial boolean into those functions so as to avoid doing unneeded work.

Reply:
Sure, good idea.

Reply:
I just used the event passed to these functions via the event listeners to determine whether it's the initial call or not (rather than creating an extra flag).

Also replaced `loadVisibleFonts()` with `filterFonts()` because we might as well avoid doing the filtering twice as well.

Reply:
Ah good idea on checking the event!